### PR TITLE
Previous regular expression was badly reading FC id in the case of MiSeq runs

### PR DESCRIPTION
**Pull request by vezzi**

### CODE REVIEW

**Comment by vezzi** [comment]
@Galithil @senthil10 this is for you

**Comment by vezzi** [comment]
ops... need to figure out why trevis is failing ... sorry

File: tests/test_parse_samplesheet.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def test_samplesheet():
 
 def test_runinfo():
     path = os.path.dirname(os.path.abspath(__file__))
-    k=classes.RunInfoParser(os.path.join(path, '../test_data/150424_ST-E00214_0031_BH2WY7CCXX/RunInfo.xml'))
+    k=classes.RunInfoParser(os.path.join(path, '../test_data/150424_ST-E00214_0031_BH2WY7CCXX/RunInfo.xml')
```
Comment (senthil10) [inline tests/test_parse_samplesheet.py]: @vezzi : `Travis` is failing because of this. The `RunInfoParser` is missing the closing ')' causes syntax error.

Comment (Galithil) [inline tests/test_parse_samplesheet.py]: missing one closing parenthesis. Nothing much.

Comment (senthil10) [inline tests/test_parse_samplesheet.py]: @Galithil : **JINX** :smile:

Comment (Galithil) [inline tests/test_parse_samplesheet.py]: aaaaw you win.

**Comment by vezzi** [comment]
@senthil10 and @Galithil I fixed the tests.

**Comment by senthil10** [comment]
Awesome :+1:

**Pull request merged**